components/org.wso2.carbon.identity.recovery/pom.xml

```diff
@@ -126,6 +130,29 @@
                             org.wso2.carbon.identity.handler.event.account.lock.exception;
                             version="${carbon.identity.account.lock.handler.imp.pkg.version.range}",
                             org.wso2.carbon.consent.mgt.core;version="${carbon.consent.mgt.version.range}",
+                            org.wso2.carbon.identity.application.authentication.framework.config;
```

Comment:
can't we use org.wso2.carbon.identity.application.authentication.framework.* instead of adding each package here?